### PR TITLE
[Images] Accept header with custom worker

### PR DESCRIPTION
**Pull request by marciocloudflare**
- Clarified that `auto`only works with Image Resizing
- Clarified what users need in order to have a custom Worker with `format:auto`
- Addresses [Issue-4541](https://github.com/cloudflare/cloudflare-docs/issues/4541)
- Addresses [PCX-3971](https://jira.cfops.it/browse/PCX-3971)

### CODE REVIEW

**Review by netgusto — APPROVED** [review]

File: content/images/_partials/_supported-properties.md [hunk 1]
```diff
@@ -204,11 +204,11 @@ Affects interpretation of `width` and `height`. All resizing modes preserve aspe
 
 #### `format`
 
-{{<Aside type="note" header="Note">}}At the moment, this setting is ignored by Cloudflare Images.{{</Aside>}}
+{{<Aside type="note" header="Note">}}At the moment, this setting only works directly with [Image Resizing URL format](/images/image-resizing/url-format/).{{</Aside>}}
```
Comment (netgusto) [inline content/images/_partials/_supported-properties.md]:
```suggestion
{{<Aside type="note" header="Note">}}At the moment, this setting only works directly with [Image Resizing](/images/image-resizing/url-format/).{{</Aside>}}
```

**Review by sejoker — APPROVED** [review]

File: content/images/_partials/_supported-properties.md [hunk 2]
```diff
@@ -226,7 +226,25 @@ format=auto
 ---
 filename: Workers
 ---
-cf: {images: {format: "auto"}}
+cf: {images: {format: "avif"}}
+```
+
+In order for the `format:auto` option to work with a custom Worker, you need to pass the `Accept` header. Refer to [this example Worker](/images/image-resizing/resize-with-workers/#an-example-worker) for a complete overview of how to set up an Image Resizing Worker.
```
Comment (pedrosousa) [inline content/images/_partials/_supported-properties.md]:
```suggestion
For the `format:auto` option to work with a custom Worker, you need to pass the `Accept` header. Refer to [this example Worker](/images/image-resizing/resize-with-workers/#an-example-worker) for a complete overview of how to set up an Image Resizing Worker.
```

File: content/images/_partials/_supported-properties.md [hunk 3]
```diff
@@ -226,7 +226,25 @@ format=auto
...
+
+```js
+---
+filename: Custom Worker for Image Resizing with `format:auto`
```
Comment (pedrosousa) [inline content/images/_partials/_supported-properties.md]:
```suggestion
header: Custom Worker for Image Resizing with `format:auto`
```
The formatting is slightly different for code block headers and filenames: filenames are rendered with a monospaced font, while headers are not. I noticed that you also used `header:` in other code blocks in this file.

I'd only use `filename:` for... filenames. :)

**Review by pedrosousa — APPROVED** [review]

File: content/images/_partials/_supported-properties.md (hunk 3 above)
Comment (marciocloudflare) [reply content/images/_partials/_supported-properties.md]: Oh. I think I'm gonna go over the document to make sure they all have `header` instead

File: content/images/_partials/_supported-properties.md [hunk 4]
```diff
@@ -217,16 +217,34 @@ Example:
...
 ```js
 ---
-filename: Workers
+header: Workers
 ---
-cf: {images: {format: "auto"}}
+cf: {images: {format: "avif"}}
+```
+
+For the `format:auto` option to work with a custom Worker, you need to pass the `Accept` header. Refer to [this example Worker](/images/image-resizing/resize-with-workers/#an-example-worker) for a complete overview of how to set up an Image Resizing Worker.
```
Comment (kornelski) [inline content/images/_partials/_supported-properties.md:232]: s/pass/parse/

Comment (marciocloudflare) [reply content/images/_partials/_supported-properties.md:232]: What do you mean @kornelski?

**Pull request merged**